src/components/VideoPlayer/Player.js

```diff
@@ -369,13 +409,8 @@ class VideoPlayer extends Component {
               onClick={e => {
                 e.preventDefault()
                 e.stopPropagation()
-                const next = {
-                  muted: !muted
-                }
-                globalState.muted = next.muted
-                globalState.instances.forEach(setter => {
-                  setter(next)
-                })
+                this.setMuted(!muted)
+                this.captureFocus()
```

Comment:
I'm not fully convinced from an accessibility/screenreader perspective whether it's good to move focus back on the video after interacting with player controls. I'd tend to leave that decision to the user.

Reply:
Thanks for this one. I removed captureFocus from all interactions.